chat-plugins/info.js

```diff
@@ -1918,7 +1920,7 @@ exports.commands = {
 	},
 	htmlboxhelp: [
 		"/htmlbox [message] - Displays a message, parsing HTML code contained.",
-		"!htmlbox [message] - Shows everyone a message, parsing HTML code contained. Requires: ~ & #",
+		"!htmlbox [message] - Shows everyone a message, parsing HTML code contained. " + Chat.require('#'),
```

Comment:
Like, this is more text than before!

One thing we could do is just `Requires: #` in the text, and then do a search and add higher ranks after it.